proposals/simd/BinarySIMD.md

```diff
@@ -27,143 +27,170 @@ The `v8x16.shuffle` instruction has 16 bytes after `simdop`.
+| `i16x8.ge_u`              |    `0x30`| -                  |
+| `i32x4.eq`                |    `0x31`| -                  |
+| `i32x4.ne`                |    `0x32`| -                  |
+| `i32x4.lt_s`              |    `0x33`| -                  |
+| `i32x4.lt_u`              |    `0x34`| -                  |
+| `i32x4.gt_s`              |    `0x35`| -                  |
+| `i32x4.gt_u`              |    `0x36`| -                  |
+| `i32x4.le_s`              |    `0x37`| -                  |
+| `i32x4.le_u`              |    `0x38`| -                  |
+| `i32x4.ge_s`              |    `0x39`| -                  |
+| `i32x4.ge_u`              |    `0x3a`| -                  |
+| -                         |    `0x3b`| -                  |
+| -                         |    `0x3c`| -                  |
+| -                         |    `0x3d`| -                  |
+| -                         |    `0x3e`| -                  |
+| -                         |    `0x3f`| -                  |
+| -                         |    `0x40`| -                  |
+| -                         |    `0x41`| -                  |
+| -                         |    `0x42`| -                  |
+| -                         |    `0x43`| -                  |
+| -                         |    `0x44`| -                  |
+| `f32x4.eq`                |    `0x45`| -                  |
+| `f32x4.ne`                |    `0x46`| -                  |
+| `f32x4.lt`                |    `0x47`| -                  |
+| `f32x4.gt`                |    `0x48`| -                  |
+| `f32x4.le`                |    `0x49`| -                  |
+| `f32x4.ge`                |    `0x4a`| -                  |
+| `f64x2.eq`                |    `0x4b`| -                  |
+| `f64x2.ne`                |    `0x4c`| -                  |
+| `f64x2.lt`                |    `0x4d`| -                  |
+| `f64x2.gt`                |    `0x4e`| -                  |
+| `f64x2.le`                |    `0x4f`| -                  |
+| `f64x2.ge`                |    `0x50`| -                  |
+| `i8x16.add`               |    `0x51`| -                  |
+| `i8x16.add_saturate_s`    |    `0x52`| -                  |
+| `i8x16.add_saturate_u`    |    `0x53`| -                  |
+| `i8x16.sub`               |    `0x54`| -                  |
+| `i8x16.sub_saturate_s`    |    `0x55`| -                  |
+| `i8x16.sub_saturate_u`    |    `0x56`| -                  |
+| `i8x16.mul`               |    `0x57`| -                  |
+| -                         |    `0x58`| -                  |
+| -                         |    `0x59`| -                  |
+| `i8x16.shl`               |    `0x5a`| -                  |
+| `i8x16.shr_s`             |    `0x5b`| -                  |
+| `i8x16.shr_u`             |    `0x5c`| -                  |
+| `i8x16.neg`               |    `0x5d`| -                  |
+| `i8x16.any_true`          |    `0x5e`| -                  |
```

Comment:
It makes sense to me to consistently adopt use v8x16 for sign agnostic operations, it's a little weird to me that we do this only for v8x16.shuffle. Thoughts?

Reply:
Also think that using `v8x16` for only shuffles is weird (see #38), but I think keeping `i8x16` and friends for sign-agnostic operations that still only make sense for integer lanes is reasonable. To counter, what do you think of changing shuffle to be `v128.shuffle`?

Reply:
Using `v8x16.shuffle` makes sense because it's a generic byte shuffle, so the name is more indicative than a `v128.shuffle`, but I don't feel strongly about which way we go here.